torch/_functorch/_aot_autograd/graph_capture.py

```diff
@@ -43,9 +45,31 @@
 aot_graphs_log = getArtifactLogger(__name__, "aot_graphs")
 
 
-def _create_graph(f, args, *, aot_config: AOTConfig) -> torch.fx.GraphModule:
+def _create_graph(
+    f,
+    args: list[torch.Tensor],
+    args_descs: Optional[
+        list[AOTInput]
+    ] = None,  # keep compat with old clients; maybe we should split into two impls
```

Comment:
I see you made aot_module_simplified pass in descs; which old clients are we talking about here?

Reply:
This function is amazingly not called by anyone else, so no BC implications